packages/atlas/atlas.config.yml

```diff
@@ -59,7 +59,7 @@ features:
       - title: Sign Up to YouTube Partner Program
         showInDashboard: false # Optional. If false the reward will be shown only on YouTube partner program landing page. Default true
         shortDescription: Connect your YouTube channels via a simple step-by-step flow and get your first reward.
-        baseAmount: 5000 # Base amount that will be multiplied by tier multiplier
+        baseAmount: 20 # Base amount that will be multiplied by tier multiplier
```

Comment:
ok, I see two problems with this: 
 - what if we will want to reintroduce `baseAmount` as joy in the future? 
 - we should explicitly say here that this value should be provided in dollars or even rename the variable to `baseAmountInUSD`. Maybe it's worth keeping `baseAmount` and setting these to null and introducing another variable - `baseAmountInUSD`?

Reply:
@drillprop I did the same, but in the benefit card, as it accepts dollar amount and joy amount. Check the changes in BenefitCard.tsx